support-workers/src/main/scala/com/gu/zuora/ZuoraSubscriptionCreator.scala

```diff
@@ -0,0 +1,68 @@
+package com.gu.zuora
```

Comment:
this file is various functionality removed and tidied up from CreateZuoraSubscription.scala main file.

Reply:
Why move it here? Just for file size reasons?

Reply:
that and it didn't seem to have any reason to be in there as it's more like detail, not really part of the view from 10000 feet enough